tests/draft-next/optional/format/time.json

```diff
@@ -187,10 +187,15 @@
                 "valid": false
             },
             {
-                "description": "no time offset",
+                "description": "no time offset #1",
```

Comment:
Let's leave this as-is, and rather than numbering test descriptions, call the new test `no time offset with second offset` as you did in your PR title.

Reply:
done

Reply:
Thanks, I suppose I meant "second fraction" and just mis-quoted your PR title, since it's not an offset, but I'll apply that on merge, this is fine now, thanks.